grp/classic.gi

```diff
@@ -403,7 +403,7 @@ BindGlobal( "Oplus45", function()
 
     # construct the group without calling 'Group'
     g := [ phi*tau2, tau*eichler*delta ];
-    g:=List(g,i->ImmutableMatrix(f,i),true);
+    g:=List(g,i->ImmutableMatrix(f,i));
```

Comment:
Yes, it worked until 814a0f6e77bbd0472619a588188c1858d5e4e6e3 broke it by making `List` validate its arguments more agressively. My guess is that the `true` in there was an accident.